deploy/helm/sumologic/templates/setup/setup-serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.sumologic.setupEnabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name:  {{ template "sumologic.fullname" . }}-setup
```

Comment:
Do we want to name all these new resources with the same name?

Reply:
yes, it's a very conventional to give resources the same name in helm as long as we don't have two or more resources of the same type with the same name